src/RoverExplorer/RoverExplorer.tsx

```diff
@@ -1,11 +1,29 @@
 import React, {FunctionComponent} from "react";
 import styles from './RoverExplorer.module.scss';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
+import { Button } from "semantic-ui-react";
+
 
 
 export const RoverExplorer: FunctionComponent = () => {
-    return (
-        <div className={styles.roverExplorer}>
-            <h2>Rover Explorer</h2>;
-        </div>
-    );
+  return (
+    <section>
+    <div className={styles.Content}>
+      <header className={styles.Header}><h1 className={styles.HeaderText}>Rover Explorer</h1></header>
+      <div className={styles.GridContainer}>
+        <Button as={Link} to="/timeline/perserverance" className={styles.GridItem1}>Perserverance</Button>
+        <Button as={Link} to="/timeline/oppertunity" className={styles.GridItem2}>Oppertunity</Button>
+        <Button as={Link} to="/timeline/spirit" className={styles.GridItem3}>Spirit</Button>
+        <Button as={Link} to="/timeline/curiosity" className={styles.GridItem4}>Curiosity</Button>
+        <Button as={Link} to="/view_all" className={styles.GridItem5}>View All</Button>
```

Comment:
I'm still confused by this...
Why are we adding another dependency for this?

Can't it just be a `Link`?

Reply:
It didn't work we needed the library to make the divs work as buttons